### PR TITLE
feat: added responsiveness

### PR DESCRIPTION
**Pull request by sks-21**
Made the layout **completely responsive**
added installation page
refactored stylesheets

### CODE REVIEW

File: colorspage/colorspage.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    flex-wrap: wrap;
+}
+
+.colors-item-title {
+    font-weight: bold;
+}
+
+.colors-item-div {
+    width: 100px;
+    height: 100px;
```
Comment (sushil-mod) [inline colorspage/colorspage.css]: use rems

**Review by sushil-mod — APPROVED** [review]
just comment about using rems instead px ,rest is good

File: colorspage/colorspage.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.fbui-component-container-x {
+    column-gap: 60px;
```
Comment (MaruthiPothuganti) [inline colorspage/colorspage.css:2]: rem can be used instead of pixels

File: stylesheets/reset.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+* {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
```
Comment (MaruthiPothuganti) [inline stylesheets/reset.css:13]: this can be combined and written as one piece

**Review by MaruthiPothuganti — COMMENTED** [review]
Good structuring , naming of Classnames and the code looks god to merge.
Suggested a couple of changes.

**Pull request merged**